activerecord/test/cases/migration_test.rb

```diff
@@ -123,6 +123,40 @@ def test_migration_version
     assert_nothing_raised { ActiveRecord::Migrator.run(:up, MIGRATIONS_ROOT + "/version_check", 20131219224947) }
   end
 
+  def test_create_table_raises_if_already_exists
+    connection = Person.connection
+    connection.drop_table :testings, if_exists: true
```

Comment:
Just noticed, whole line should be replaced with `force` for `create_table` and same in the next test

Reply:
:+1: